testClientCertAuthentication is flaky

We should look into this failure @martinpitt 

https://logs-https-cockpit.apps.ci.centos.org/logs/pull-13263-20191210-091010-3b6ad39c-cockpit-project-cockpit--ubuntu-stable/log.html#184-2

```
WARNING: conflicting time&date synchronization service 'ntp' will be disabled in favor of chronyd


Using default chrony configuration.
 * /usr/sbin/update-rc.d sssd enable
 * /usr/sbin/service sssd restart
 * Successfully enrolled machine in realm
Journal extracted to TestRealms-testClientCertAuthentication-ubuntu-stable-127.0.0.2-2801-FAIL.log
Journal extracted to TestRealms-testClientCertAuthentication-services-127.0.0.2-2802-FAIL.log
Traceback (most recent call last):
  File "test/verify/check-realms", line 471, in testClientCertAuthentication
    session_leader='cockpit-session')
  File "test/verify/check-realms", line 445, in do_test
    self.assertEqual(len(sessions), 1)
AssertionError: 2 != 1

not ok 184 testClientCertAuthentication (check_realms.TestRealms) # duration: 155s

```

--- a/test/verify/check-realms
+++ b/test/verify/check-realms
@@ -449,6 +449,8 @@ for s in key p12 pem; do docker cp freeipa:jane.$s .; done
                 self.assertIn('cockpit; type web', out)
                 # sessions time out after 10s, but let's not wait for that
                 m.execute('loginctl terminate-session ' + sessions[0])
+                # wait until the session is gone
+                m.execute("while loginctl show-user jane; do sleep 1; done")
 
             m.stop_cockpit()
 

